README.md

```diff
@@ -4,48 +4,50 @@ Add [Ceedling](https://github.com/ThrowTheSwitch/Ceedling) task for analyzing
 code with [Cppcheck](http://cppcheck.net/).
 
 <!-- TOC ignore:true -->
+
 ## Contents
 
 <!-- TOC -->
 
 - [Installation](#installation)
 - [Enable the plugin](#enable-the-plugin)
 - [Configuration](#configuration)
-	- [Reports](#reports)
-		- [Text](#text)
-		- [XML](#xml)
-		- [HTML](#html)
-	- [Preprocessor defines](#preprocessor-defines)
-		- [Define](#define)
-		- [Undefine](#undefine)
-	- [Includes](#includes)
-	- [Excludes](#excludes)
-	- [Platform](#platform)
-	- [Standard](#standard)
-	- [Check Level](#check-level)
-	- [Addons](#addons)
-		- [MISRA with rule texts file](#misra-with-rule-texts-file)
-	- [Checks](#checks)
-	- [Suppressions](#suppressions)
-		- [Inline](#inline)
-		- [List Files](#list-files)
-		- [Command Line](#command-line)
-	- [Library configuration](#library-configuration)
-	- [Rules](#rules)
-	- [Extra arguments](#extra-arguments)
+  - [Reports](#reports)
+    - [Text](#text)
+    - [XML](#xml)
+    - [HTML](#html)
+  - [Project](#importing-project)
+  - [Preprocessor defines](#preprocessor-defines)
+    - [Define](#define)
+    - [Undefine](#undefine)
+  - [Includes](#includes)
+  - [Excludes](#excludes)
+  - [Platform](#platform)
+  - [Standard](#standard)
+  - [Check Level](#check-level)
+  - [Addons](#addons)
+    - [MISRA with rule texts file](#misra-with-rule-texts-file)
+  - [Checks](#checks)
+  - [Suppressions](#suppressions)
+    - [Inline](#inline)
+    - [List Files](#list-files)
+    - [Command Line](#command-line)
+  - [Library configuration](#library-configuration)
+  - [Rules](#rules)
+  - [Extra arguments](#extra-arguments)
 - [Usage](#usage)
-	- [Analyze whole project](#analyze-whole-project)
-	- [Analyze single file](#analyze-single-file)
+  - [Analyze whole project](#analyze-whole-project)
+  - [Analyze single file](#analyze-single-file)
 
 <!-- /TOC -->
 
 ## Installation
 
 Clone this into Ceedling's plugin folder of your current project.
 
-```shell
-$ cd <your-project>/vendor/ceedling/plugins
-$ git clone https://github.com/deltalejo/cppcheck-ceedling-plugin.git cppcheck
+```bash
```

Comment:
Any reason why `bash` instead of the more generic `shell`?

Reply:
No, I just noticed that `bash` was used in some places in other `shell`, I just wanted to keep the same pattern.

Reply:
Let's keep all as `shell`.